mesh/model.c

```diff
@@ -856,23 +858,23 @@ static void send_dbus_dev_key_msg_rcvd(struct mesh_node *node, uint8_t ele_idx,
 }
 
 static void send_dev_key_msg_rcvd(struct mesh_node *node, uint8_t ele_idx,
-					   uint16_t src, uint16_t app_idx,
-					   uint16_t net_idx, uint16_t size,
-					   const uint8_t *data)
+					uint16_t src, uint16_t dst,
+					uint16_t app_idx, uint16_t net_idx,
+					uint16_t size, const uint8_t *data)
 {
 	struct l_io *io = node_get_fd_io(node);
 	struct mesh_amqp *amqp = node_get_amqp(node);
 
 	if (amqp && mesh_amqp_is_ready(amqp))
-		send_fd_dev_key_msg_rcvd(ele_idx, src, app_idx, net_idx,
-				 size, data, send_amqp, amqp);
+		send_fd_dev_key_msg_rcvd(ele_idx, src, dst, app_idx, net_idx,
+						size, data, send_amqp, amqp);
 
 	if (io)
-		send_fd_dev_key_msg_rcvd(ele_idx, src, app_idx, net_idx,
-					 size, data, send_fd, io);
+		send_fd_dev_key_msg_rcvd(ele_idx, src, dst, app_idx, net_idx,
```

Comment:
`send_fd_dev_key_msg_rcvd(ele_idx, src, node_get_primary(node) + ele_idx, ...`

Reply:
? I don't understand. I got `dst` and I just want to pass it. Why to duplicate some logic that computes `dst` address. It simple and for `AppKey` messages it is solved in the same way.